services/metric/measurement.go

```diff
@@ -48,8 +48,8 @@ func (r pendingEventsMeasurement) GetName() string {
 
 func (r pendingEventsMeasurement) GetTags() map[string]string {
 	res := map[string]string{
-		"workspace": r.workspace,
```

Comment:
Please retain the old tag as well until we have time to adapt the relevant alerts, e.g. [here](https://github.com/rudderlabs/rudderstack-operator/blob/master/helm-charts/rudderstack-monitor/charts/prometheus/alerts/core/negative-jobsdb-rt-pending-event-count.yaml#L1), [here](https://github.com/rudderlabs/rudderstack-operator/blob/master/helm-charts/rudderstack-monitor/charts/prometheus/alerts/core/jobsdb-batch-rt-pending-event-count.yaml#L1), [here](https://github.com/rudderlabs/rudderstack-operator/blob/master/helm-charts/rudderstack-monitor/charts/kapacitor/alerts/tf-proxy-input-output-diff-count.yaml#L17)

We can remove the deprecated tag during the next release

Reply:
done